apps/backend/src/handlers/validationHandler.ts

```diff
@@ -19,10 +23,18 @@ export const validationHandler: CustomHandler<{
 
 	const user = await clerkClient.users.getUser(userId);
 
-	// TODO: return orgs
+	if (user.username === null) {
+		throw new UnauthorizedError();
+	}
+
+	const userOrgs = await clerkClient.users.getOrganizationMembershipList({
```

Comment:
We can incapsulate all logic related to clerk in dedicated service later

Reply:
https://linear.app/codemod/issue/CDMD-3094/add-clerkservice-in-backend @DmytroHryshyn